modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/flamingos2/Flamingos2ControllerSim.scala

```diff
@@ -50,11 +48,9 @@ final case class Flamingos2ControllerSimBad(failAt: Int) extends Flamingos2Contr
 
   private val counter: AtomicInteger = new AtomicInteger(0)
 
-  override def observe(obsid: ImageFileId, expTime: Time): SeqAction[ImageFileId] = EitherT( Task {
-    Log.info("Taking Flamingos-2 observation with label " + obsid)
```

Comment:
`Flamingos2ControllerSimBad` should put a log message to indicate that we are using a simulated F2.